### DISCUSSION TITLE
Volume control

### DISCUSSION THREAD

**Discussion by Jakes9** [mikebrady/shairport-sync#1659] [Q&A]
Hello is this possible to control volume in iPhone from linux device?
I have Bluetooth speaker with volume buttons. When i change volume in iPhone it's changes in alsamixer and over a2dp in Bluetooth speaker. Is it possible to controlling it from speaker to iPhone ?

**Comment by mikebrady** [comment]
Hi there. This is a good question -- it has been asked and answered a few times. Unfortunately, it is not really possible in any reliable or generalisable way. You should find more discussion about it in some of the closed issues.
